README.md

```diff
@@ -5,14 +5,312 @@
 This repository contains the Instant Motion Tracking project
 using MediaPipe.
 
-## Source Code Headers
+This document focuses on the [below graph](#main-graph) that performs instant motion tracking.
 
-Every file containing source code must include copyright and license
-information. This includes any JS/CSS files that you might be serving out to
-browsers. (This is to help well-intentioned people avoid accidental copying that
-doesn't comply with the license.)
+For overall context on instant motion tracking, please read this [Google Developers Blog - TODO](#).
 
-Apache header:
+![instant_motion_tracking_android_gif1](images/demo_1.gif) ![instant_motion_tracking_android_gif2](images/demo_2.gif)
+
+## Android
+
+[Source](mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking)
+
+To build and install the app, the MediaPipe repository must first be cloned from Github:
+
+```bash
+git clone https://github.com/google/mediapipe.git
+```
+
+Once you have the MediaPipe repository, clone this project as well:
+
+```bash
+git clone https://github.com/googleinterns/instant_motion_tracking.git
+```
+
+Due to BUILD dependency issues with related MediaPipe projects, you must change the visibility permissions of the object_tracking_3d calculators. Run the command below to copy the modified BUILD file to fix privacy issues between projects:
+
+```base
+mv instant_motion_tracking/mediapipe/graphs/instantmotiontracking/calculators/obj3d/BUILD mediapipe/mediapipe/graphs/object_detection_3d/calculators/BUILD
+```
+
+Next, run the following command to move the instantmotiontracking source graphs and calculators into the MediaPipe directory:
+
+```bash
+mv instant_motion_tracking/mediapipe/graphs/instantmotiontracking mediapipe/mediapipe/graphs/instantmotiontracking
+```
+
+It is also necessary move the Android source application to Mediapipe:
+
+```bash
+mv instant_motion_tracking/mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking mediapipe/mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking
+```
+
+Now, navigate to the MediaPipe directory and run the following command to build the application with Bazel
+
+```bash
+bazel build -c opt --config=android_arm64 mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking
+```
+
+Once the app is built, install it on an Android device with:
+
+```bash
+adb install bazel-bin/mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/instantmotiontracking.apk
+```
+
+## Graph
```

Comment:
Can you use https://viz.mediapipe.dev/ to generate graph visualization instead of pasting the graph txt?

Reply:
I've duplicated the format from other MediaPipe projects, so the current README includes both the visualizations of the graph from viz.mediapipe.dev and the graph code itself (pbtxt). However, I can remove the pbtxt if that's preferable, please let me know. (Visualizations of the each graph can be found in [images/](https://github.com/googleinterns/instant_motion_tracking/tree/object-rendering/images))

Reply:
No worries then. Just remember to update here if you change the graph. Might be a good idea to do it when you are done, otherwise you have to update here multiple times.

Reply:
Will be sure to add a note regarding changes once the system is completed (README + Installation Instructions if they change)